src/components/Layout/Footer.tsx

```diff
@@ -290,6 +290,7 @@ export function Footer() {
           </FooterLink>
           <FooterLink href="/learn/">Quick Start</FooterLink>
           <FooterLink href="/learn/installation">Installation</FooterLink>
+          <FooterLink href="/learn/typescript">Using TypeScript</FooterLink>
```

Comment:
Let's remove this, the links here are for the top-level grouping, and "Using Typescript" is under "Installation" listed above.